src/components/PDFView/index.native.js

```diff
@@ -155,7 +155,7 @@ class PDFView extends Component {
                         fitPolicy={0}
                         trustAllCerts={false}
                         renderActivityIndicator={() => <FullScreenLoadingIndicator />}
-                        source={{uri: this.props.sourceURL}}
+                        source={{uri: this.props.sourceURL, cache: true, expiration: 864000}}
```

Comment:
@tienifr I think we should raise a discussion for the expiration interval. IMO 12 hours should be good.

Reply:
another question: will the cache be removed when attachment is deleted from the chat?

Reply:
>Seems like it does not. But it still gets removed when we logged out.

We should clear the cache when the attachment is deleted, and i am not sure how this will work incase of protected pdf i am checking.